__tests__/lib/openapi-to-json-schema.test.ts

```diff
@@ -455,33 +490,35 @@ describe('`format` support', () => {
       ['integer', 'uint64', 0, 2 ** 64 - 1], // 0 to 1844674407370955161
       ['number', 'float', 0 - 2 ** 128, 2 ** 128 - 1], // -3.402823669209385e+38 to 3.402823669209385e+38
       ['number', 'double', 0 - Number.MAX_VALUE, Number.MAX_VALUE],
-    ])('`%s`', (type, format, min, max) => {
-      it('should add a `minimum` and `maximum` if not present', () => {
-        expect(toJSONSchema({ type: type as JSONSchema7TypeName, format })).toStrictEqual({
-          type,
-          format,
-          minimum: min,
-          maximum: max,
+    ])('`type: %s`', (type, format, min, max) => {
+      describe(`\`format: ${format}\``, () => {
```

Comment:
Changed this because we were duping all of these integer + format tests under the same `describe` block and they weren't properly exposing themselves in the test output. Thankfully they were still being run, it just wasn't clear what was being run for what.